pages/kids/memory-game/index.tsx

```diff
@@ -1,29 +1,50 @@
 import { useTranslation } from 'next-i18next';
-import React from 'react';
+import React, { useMemo } from 'react';
 import dynamic from 'next/dynamic';
 import SEO from 'components/basecomponents/SEO';
 import { getCountryVariant } from 'utils/locales';
-export { getStaticProps } from 'utils/localization';
+import { GetStaticProps, InferGetStaticPropsType } from 'next';
+import { DictionaryDataObject, fetchDictionary, getKidsCategory } from '../../../utils/getDataUtils';
+import { getServerSideTranslations } from '../../../utils/localization';
+import { PhrasesContext } from 'components/basecomponents/MemoryGame/PhrasesContext';
 
 const MemoryGame = dynamic(() => import('components/basecomponents/MemoryGame/MemoryGameThemeLoader'), {
   ssr: false,
 });
 
-const MemoryGameSection = () => {
+const MemoryGameSection = ({ dictionary }: InferGetStaticPropsType<typeof getStaticProps>) => {
   const { t } = useTranslation();
+  const kidsCategory = useMemo(() => getKidsCategory(dictionary), [dictionary]);
+  if (!kidsCategory) {
+    return null;
+  }
 
   return (
-    <div className="bg-gradient-to-r from-[#fdf6d2] to-[#99bde4] -mb-8 -m-2">
-      <SEO
-        title={t(`seo.kids_page_memorygame_title.${getCountryVariant()}`)}
-        description={t(`seo.kids_page_memorygame_description.${getCountryVariant()}`)}
-        image="https://www.movapp.cz/icons/movapp-cover-kids.jpg"
-      />
-      <div className="flex flex-wrap flex-col items-center min-h-screen m-auto sm:py-10 py-2 px-2 sm:px-4 overflow-hidden">
-        <MemoryGame />
+    <PhrasesContext.Provider value={kidsCategory.translations}>
```

Comment:
Why do you need `Context` here? Was it not possible to just pass the `kidsCategory.translations` to `<MemoryGame>` ?

Reply:
If its harder than it looks, don't worry, we can simplifiy it later but it's worth a shot :)

Reply:
I would prefer props too. I know that data goes through two another components ,but it has a simple reason -> Loading only specific css file for each theme. If it was possible to lazy load css module we can move code from Theme components to ThemeLoader.

Reply:
I will change it, but I think it's a props drilling)

Reply:
Yeah we need to refactor the ThemeLoader in the future, for now prop-drilling through 2 components is more readable than Context I think :)